fonts/README.md

```diff
@@ -0,0 +1,15 @@
+# Fonts
+
+Sigma-9 self-hosts several fonts to reduce external depency and minimize `@import`, which are render-blocking resources and potentially costly in terms of performance.
+
+## [Inter](https://github.com/rsms/inter)
+
+Sigma-9's body font. Licensed under [SIL Open Font License 1.1](https://github.com/rsms/inter/blob/master/LICENSE.txt).
+
+## [Sans Normalcy](https://github.com/rsms/inter)
```

Comment:
Link is incorrect 

```suggestion
 ## [Sans Normalcy](https://github.com/scpwiki/normalcy-sans)
```

Reply:
Unrelated: I didn't realize it was "Sans Normalcy" not "Normalcy Sans" (like Noto Sans). Should we move the repo?

Reply:
I think it's fine both ways - Sans Normalcy is the punny way of reading it, Normalcy Sans can be a more traditional alt-name.